docs/getting-started/endpoint-data-volume.asciidoc

```diff
@@ -1,12 +1,58 @@
 [[endpoint-data-volume]]
-= Configure {elastic-endpoint}'s data volume
+= Configure data volume for {elastic-endpoint}
 ++++
 <titleabbrev>Configure data volume</titleabbrev>
 ++++
 
+{elastic-endpoint}, the installed component that performs {elastic-defend}'s threat monitoring and prevention, has been optimized to reduce data volume and CPU usage. You can disable or modify some of these optimizations by reconfiguring the following <<adv-policy-settings,advanced settings>> in the {elastic-defend} integration policy.
 
-[sidebar]
---
-[.text-center]
-**This is a placeholder for future documentation.**
---
+IMPORTANT: Modifying these advanced settings from their defaults will increase the volume of data that {elastic-endpoint} processes and ingests, and increase {elastic-endpoint}'s CPU usage. Make sure you're aware of how these changes will affect your storage capabilities and performance.
+
+Each setting has several OS-specific variants, represented by `[linux|mac|windows]` in the names listed below. Use the variant relevant to your hosts' operating system (for example, `windows.advanced.events.deduplicate_network_events` to configure network event deduplication for Windows hosts).
+
+[discrete]
+[[network-event-deduplication]]
+== Network event deduplication
+
+added:[8.15] When repeated network connections are detected from the same process, {elastic-endpoint} will not produce network events for subsequent connections. To disable or reduce deduplication of network events, use these advanced settings:
+
+`[linux|mac|windows].advanced.events.deduplicate_network_events`:: Enter `false` to completely disable network event deduplication. Default: `true`
+
+`[linux|mac|windows].advanced.events.deduplicate_network_events_below_bytes`:: Enter a transfer size threshold (in bytes) for events you want to deduplicate. Connections below the threshold are deduplicated, and connections above it are not deduplicated. This allows you to suppress repeated connections for smaller data transfers but always generate events for larger transfers. Default: `1048576` (1MB)
+
+[discrete]
+[[host-fields]]
+== Data in `host.*` fields
+
+added:[8.18] {elastic-endpoint} includes only a small subset of the data in the `host.*` fieldset in event documents. Full `host.*` information is still included in documents written to the `metrics-*` index pattern and in {elastic-endpoint} alerts. To override this behavior and include all `host.*` data for events, use this advanced setting:
+
+`[linux|mac|windows].advanced.set_extended_host_information`:: Enter `true` to include all `host.*` event data. Default: `false`
+
+NOTE: Users should take note of how a lack of some `host.*` information may affect their <<event-filters,event filters>> or alert exceptions.
```

Comment:
In case users want to learn more about Elastic Endpoint exceptions. 

```suggestion
NOTE: Users should take note of how a lack of some `host.*` information may affect their <<event-filters,event filters>> or <<endpoint-rule-exceptions,Endpoint alert exceptions>>.
```